widget/list.go

```diff
@@ -20,6 +20,7 @@ type List struct {
 	scrollContainerOpts      []ScrollContainerOpt
 	sliderOpts               []SliderOpt
 	entries                  []interface{}
+	entriesTable             map[interface{}]*Button
```

Comment:
This map needs to be initialized ~ line 87. When I run the list demo it fails to start due to this map being null

```
	l := &List{
		EntrySelectedEvent: &event.Event{},

		init:           &MultiOnce{},
		focusIndex:     0,
		prevFocusIndex: -1,
		entriesTable:   make(map[interface{}]*Button),
	}

```